tests/jquery.flot.logaxis.Test.js

```diff
@@ -141,6 +141,22 @@ describe("unit tests for the log scale functions", function() {
                     }
             });
     });
+
+    it('should set axis.max to not be less then axis.min', function() {
+        var plot = $.plot(placeholder, [[0, 1, 2, 3]], {
+                xaxes: [{
+                    mode: 'log',
+                    offset : { below: -1, above: -1 },
+                    max: 1
+                }]
+            }),
+            axis, ticks,
+
+        axis = plot.getAxes().xaxis;
+
+        expect(axis.min).toBe(0.1);
+        expect(axis.max).toBe(1);
```

Comment:
I think there should be 2 tests: one for expect(axis.min).toBeSmallerThan(axis.max), and a different one for the "0.1 constant" -> this is a small feature and it deserves its own test.